skbio/sequence/_rna.py

```diff
@@ -118,13 +128,13 @@ def complement_map(cls):
 
     @classproperty
     @stable(as_of="0.4.0")
```

Comment:
remove this line (`stable`)

also remove `stable` line from `complement_map` above (I can't comment on the specific line b/c it's not included in the diff)